xrpl4j-address-codec/src/test/java/org/xrpl/xrpl4j/codec/addresses/XAddressTest.java

```diff
@@ -178,7 +177,7 @@ public void convertBetweenClassicAndXAddressMainnet(
       .test(false)
       .build();
 
-    assertThat(fromXAddress).isEqualTo(classicAddress);
+    Assertions.assertThat(fromXAddress).isEqualTo(classicAddress);
```

Comment:
Can we unwind all the changes that add `Assertions.` - they're just adding noise, and are out of step with our style through most of the other tests.